identity/manager.go

```diff
@@ -95,13 +100,99 @@ func (m *Manager) Create(ctx context.Context, i *Identity, opts ...ManagerOption
 	}
 
 	if err := m.r.PrivilegedIdentityPool().CreateIdentity(ctx, i); err != nil {
+		if errors.Is(err, sqlcon.ErrUniqueViolation) {
+			return m.generateCredentialHint(ctx, err, i)
+		}
 		return err
 	}
 
 	trace.SpanFromContext(ctx).AddEvent(events.NewIdentityCreated(ctx, i.ID))
 	return nil
 }
 
+func (m *Manager) generateCredentialHint(ctx context.Context, e error, i *Identity) (err error) {
+	var hints []string
+	var message string
+	for _, va := range i.VerifiableAddresses {
```

Comment:
hmm yeah this shouldn't be relied upon. The verifiable addresses can also be nil and if the identity schema does not setup the account verification schema extension, then it won't work. 

A better approach is to get all the credentials that exist on this account through its identifier (which you already have). It might be an expensive task and can lead to exposing sensitive account information (as @zepatrik mentioned).

I would make it configurable.